File: exercises/day11-AWorldWithoutMock/java/src/main/java/domain/Toy.java
package domain;

public class Toy {

    public enum State {
        UNASSIGNED,
        IN_PRODUCTION
    }

    private final String name;
    private State state;

    public Toy(String name, State state) {
        this.name = name;
        this.state = state;
    }

    public String getName() {
        return name;
    }

    public State getState() {
        return state;
    }

    public void setState(State newState) {
        this.state = newState;
    }
}

File: exercises/day11-AWorldWithoutMock/csharp/AWorldWithoutMock/Toy.cs
namespace AWorldWithoutMocksBefore
{
    public enum ToyState
    {
        Unassigned,
        InProduction
    }

    public class Toy
    {
        public string Name { get; }
        public ToyState State { get; set; }

        public Toy(string name, ToyState state)
        {
            Name = name;
            State = state;
        }
    }
}
